### PR TITLE
add support for validation rules

### LINKED ISSUE #82: allow validation rules to be passed during schema execution
validation rules are the recommended way to implement things like depth limit validators and other sorts of protectors.
When inspecting the source code, I found out that `validation_rules` aren't passed as an argument during schema execution. It would be nice if this could be fixed so that we could implement our own validation rules.

https://github.com/graphql-python/graphql-server/blob/c03e1a4177233b0a053948c96ce862314d52e7bf/graphql_server/flask/graphqlview.py#L87-L98

over here, the http query is run via the `run_http_query` function.
This method also takes other execution options, as shown below.

https://github.com/graphql-python/graphql-server/blob/c03e1a4177233b0a053948c96ce862314d52e7bf/graphql_server/__init__.py#L58-L67

Inside the `run_http_query` method, this function is called.

https://github.com/graphql-python/graphql-server/blob/c03e1a4177233b0a053948c96ce862314d52e7bf/graphql_server/__init__.py#L122-L124

this function, `get_response` takes another execution option, `validation_rules`.
This is not being passed anywhere, and always remains None.

https://github.com/graphql-python/graphql-server/blob/c03e1a4177233b0a053948c96ce862314d52e7bf/graphql_server/__init__.py#L234-L243

I will send a PR ASAP!

### PR DESCRIPTION
**Pull request by codebyaryan**
allows passing validation rules as an argument to the graphqlview.
also updated docs regarding the same. fixes #82

### CODE REVIEW

**Comment by codebyaryan** [comment]
oops, committed venv by mistake. Updated .gitignore and removed the dir.

File: graphql_server/aiohttp/graphqlview.py [hunk 1]
```diff
@@ -75,6 +76,9 @@ def get_context(self, request):
     def get_middleware(self):
         return self.middleware
 
+    def get_validation_rules(self):
+        return self.validation_rules
```
Comment (jkimbo) [inline graphql_server/aiohttp/graphqlview.py:80]: This should return the default validation rules if `validation_rules` is `None`:

`from graphql import specified_rules`

Comment (codebyaryan) [reply graphql_server/aiohttp/graphqlview.py:80]: okay! I'll make another commit!

Comment (codebyaryan) [reply graphql_server/aiohttp/graphqlview.py:80]: @jkimbo this is already handled at the core, though. Do you think that this is necessary?

https://github.com/graphql-python/graphql-core/blob/cc303671559822d9c3991e04fee170dce0f23553/src/graphql/validation/validate.py#L49-L50

**Review by jkimbo — DISMISSED** [review]

**Comment by codebyaryan** [comment]
When would this be merged?🤔

**Comment by jkimbo** [comment]
@codebyaryan I just merged https://github.com/graphql-python/graphql-server/pull/84 Could you bring this branch up to date?

**Comment by codebyaryan** [comment]
@jkimbo done!

**Pull request merged**